packages/theme/src/theme/_buttons.scss

```
@@ -59,6 +62,7 @@
 	&-info {
 		@include btn-colors($btn-info-bg);
 		@include btn-inverse-colors($btn-info-bg);
+		@include btn-tertiary-colors($btn-tertiary-info-color);
```

```suggestion
		@include btn-tertiary-colors($btn-info-bg);
```
Donc define another variable for the same info color. Use the same as the other buttons

---

updated